### PR TITLE
Allow arbitrary property and mapping files

### LINKED ISSUE #13: Allow taxel.yaml to be in another place
in [PR12](https://github.com/ubi-agni/human_hand/pull/12#issuecomment-1253007911) I started to think of splitting the URDF hand part from the tactile part but the concern was raised that this split might not be trivial or hinder future possibilities.

Let suppose we keep URDF hand part and tactile part together, would you mind to allow the `taxel.yaml` to be loaded from a different package/path ?

The file is almost fully hardcoded to be _human_hand_description/model/taxel.yaml_.

https://github.com/ubi-agni/human_hand/blob/0acf44b821b3408c0d909b0d96bb7b6a7273acb9/model/human_hand.urdf.xacro#L142

passing the `package` property through command line would be one step, but the path would still enforce the `taxel.yaml` file to be in a `model` subfolder of the package folder. So maybe passing a `taxel_path` property is easier,  with a default value  `arg package + "/models"`

Our goal is to have all our configuration files (not only taxel.yaml) in a single folder and have adequate launch commands passing the correct files. Hence not requiring to overwrite `taxel.yaml` of `human_hand_description` anymore.

What are your thoughts on that ? I think the change can be easy and I could try and provide a PR

### PR DESCRIPTION
**Pull request by rhaschke**
Fixes #13. Alternative to #14.

### CODE REVIEW

File: launch/upload.launch [hunk 1]
```diff
@@ -15,10 +15,12 @@
...
-	<param name="robot_description" if="$(arg __noetic)" command="xacro $(find human_hand_description)/robots/human_hand.urdf.xacro use_synergy:=$(arg use_synergy) property_file:=$(arg property_file) mapping_file:=$(arg mapping_file) tactile_mapping:=$(arg tactile_mapping) tactile_channel:='$(arg tactile_channel)' prefix:=$(arg prefix) scale:=$(arg scale) side:=$(arg side)" />
-	<param name="robot_description" unless="$(arg __noetic)" command="xacro --xacro-ns $(find human_hand_description)/robots/human_hand.urdf.xacro use_synergy:=$(arg use_synergy) property_file:=$(arg property_file) mapping_file:=$(arg mapping_file) tactile_mapping:=$(arg tactile_mapping) tactile_channel:='$(arg tactile_channel)' prefix:=$(arg prefix) scale:=$(arg scale) side:=$(arg side)" />
+	<param name="robot_description" command="xacro $(arg __extra_args) $(find human_hand_description)/robots/human_hand.urdf.xacro use_synergy:=$(arg use_synergy) property_file:=$(arg property_file) mapping_file:=$(arg mapping_file) tactile_mapping:=$(arg tactile_mapping) tactile_channel:='$(arg tactile_channel)' prefix:=$(arg prefix) scale:=$(arg scale) side:=$(arg side)"/>
```
Comment (guihomework) [inline launch/upload.launch:23]: From the visual diff in github, I see the `mapping_file` and `property_files` are already present in this separate commit

Comment (rhaschke) [reply launch/upload.launch:23]: Yes, of course. What is your point?

Comment (guihomework) [reply launch/upload.launch:23]: it just seemed strange to have a separate commit to simplify robot_description upload and already mix it with the addition of the mapping file that occurs in the third commit. I sometimes also intend to nicely split commits (atomic commits) but make mistakes and include unwanted code. It was just to make you notice it.

Comment (rhaschke) [reply launch/upload.launch:23]: The mapping_file and property_file were introduced in the first commit, while the last one does the simplification.
As we should keep all three commits (to maintain authorship), I don't see an issue here.

Comment (guihomework) [reply launch/upload.launch:23]: I apologized, I got confused about the order of commits in the github web interface for PR (not using it enough apparently).


For me github had a timeline in reverse commit (order) most recent top a seen on the main commit list 
Screenshot 2022-10-19 at 11-41-57 Commits · ubi-agni human_hand

but the PR does it reversed

Screenshot 2022-10-19 at 11-42-31 Allow arbitrary property and mapping files by rhaschke · Pull Request #15

For my defense, I should mention that there was no 4th commit yet, that could have helped me see the timeline is reversed with the date going up when going down the line

I saw the simplification as first (bottom) of the timeline, and made an interpretation from there. 

Thanks for making me discover how github is not consistent within its interfaces.

**Pull request merged**